mava/systems/ff_ippo.py

```diff
@@ -517,10 +514,12 @@ def run_experiment(_config: DictConfig) -> None:
 
         # Log the results of the training.
         elapsed_time = time.time() - start_time
-        learner_output.episodes_info["steps_per_second"] = steps_per_rollout / elapsed_time
-        logger.log_trainer_metrics(
-            experiment_output=learner_output, t_env=steps_per_rollout * (i + 1)
-        )
+        t = steps_per_rollout * (eval_step + 1)
+        learner_output.episode_metrics["steps_per_second"] = steps_per_rollout / elapsed_time
+        learner_output.episode_metrics["timestep"] = t
+
+        logger.log(learner_output.episode_metrics, t, eval_step, LogEvent.ACT)
```

Comment:
How do you feel about named arguments here (and all the `logger.log` calls) too? I realise it can get messy, but I am a big fan in almost all situations.

Reply:
hmm, but looking at it again, perhaps it is too messy.

Reply:
Agreed, happy with it in most cases, especially when args aren't clear, but in this case it's too messy to be worth it. Others I am happy with :smile: